oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/BearerTokenAuthenticationFilter.java

```diff
@@ -139,10 +134,12 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 	 * Set the {@link BearerTokenResolver} to use. Defaults to
 	 * {@link DefaultBearerTokenResolver}.
 	 * @param bearerTokenResolver the {@code BearerTokenResolver} to use
+	 * @deprecated Instead, use {@link BearerTokenAuthenticationConverter} explicitly
+	 * @see BearerTokenAuthenticationConverter
 	 */
 	public void setBearerTokenResolver(BearerTokenResolver bearerTokenResolver) {
```

Comment:
Will you please add a `@Deprecated` annotation as well?

Reply:
Sure, I will re-request review soon.